### PR TITLE
Update deprecated chart template usage

### PR DESCRIPTION
**Pull request by doncem**
### Change description ###

Usage of `values.template.yaml` is deprecated and TTL is until the week after

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/rpe-send-letter-service/values.preview.template.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+java:
```
Comment (nitinprabhu) [inline charts/rpe-send-letter-service/values.preview.template.yaml]: are we sure we don't need any env config here ?

File: charts/rpe-send-letter-service/values.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    FTP_PORT: "22"
+    FTP_FINGERPRINT: "SHA256:3tX3DIkqd1Loz2alHfnt+qjHocfxk0YUOZHlnf9Zgdk"
+    FTP_TARGET_FOLDER: "TO_XEROX"
+    FTP_REPORTS_FOLDER: "FROM_XEROX"
+    FTP_DOWNTIME_FROM: "23:58"
+    FTP_DOWNTIME_TO: "23:59"
+    # mails
+    SPRING_MAIL_HOST: "false"
+  # Don't modify below here
+  image: ${IMAGE_NAME}
```
Comment (doncem) [inline charts/rpe-send-letter-service/values.yaml]: shouldn't be hardcoded: https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L12

File: charts/rpe-send-letter-service/values.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  ingressHost: ${SERVICE_FQDN}
```
Comment (doncem) [inline charts/rpe-send-letter-service/values.yaml]: shouldn't be set by default:
```suggestion
```

**Comment by doncem** [comment]
will work a bit later on this

**Pull request closed without merging**